### ISSUE TITLE
Support for Format Strings?

### ISSUE THREAD

**Issue by irrationalistic** [atom/first-mate#43]
Right now it looks like upcase and downcase are supported, but what about regular expressions? Can't seem to get them to work! Something like `${1/[ ]/-/g}` should replace all spaces in `$1` with dashes. Is this supported and I'm doing it wrong?

**Comment by kevinsawicki** [comment]
> Right now it looks like upcase and downcase are supported,

Yeah, these are the only transformations supported right now, others need to be implemented.
